distribution/docs/src/main/resources/_contents/installing-contents.adoc

```diff
@@ -56,250 +54,216 @@ echo $JAVA_HOME
-----
+Set directory permissions to protect the ${branding} from unauthorized access.
 
-.Windows
-----
-<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}.bat
-----
+. Right-click on the file or directory noted below then select *Full Control -> Administrators -> System*.
+. Click *Properties -> Security -> Advanced* and select `Creator Owner` for `<${branding}_HOME>` (e.g., `C:\${branding-lowercase}`).
+. Restrict access to sensitive files by ensuring that only *System* and *Administrators* have Full Control to the below files by right-clicking on the file or directory below then selecting *Properties -> Security -> Advanced*
+. Delete any other groups or users listed with access to `<${branding}_HOME>/etc` and `<${branding}_HOME>/deploy`.
 
-The distribution takes a few moments to load depending on the hardware configuration.
-Execute the following command at the command line for status:
+===== Directory Permissions on *NIX
 
-.View Status
-----
-${branding}${at-symbol}local>list
-----
-The list of bundles should look similar to this:
+Set directory permissions to protect the ${branding} from unauthorized access.
 
-.${branding} List of Apps Installed
-----
-.${branding}${at-symbol}local>list
-START LEVEL 100 , List Threshold: 50
-   ID   State         Blueprint      Spring    Level  Name
-[ 111] [Active     ] [            ] [       ] [   80] Commons IO (2.1.0)
-[ 113] [Active     ] [Created     ] [       ] [   80] ${branding} :: Distribution :: Console Branding Plugin (${project.version})
-----
+* Change ownership of ${branding}_HOME
+** `chown -R NON_ROOT_USER <${branding}_HOME>`
+* Change group ownership on sub-directories
+** `chgrp -R ${branding}_GROUP <${branding}_HOME>/etc <${branding}_HOME>/data <${branding}_HOME>/deploy`
+* Change group permissions
+** `chmod -R g-w <${branding}_HOME>/etc <${branding}_HOME>/data <${branding}_HOME>/deploy`
+* Remove permissions for users
+** `chmod -R o-rwx <${branding}_HOME>/etc <${branding}_HOME>/data <${branding}_HOME>/deploy`
 
-.${branding} Application Installation Dependencies
 
-[WARNING]
+[IMPORTANT]
+====
+The system administrator must restrict certain directories to ensure that the application (user) cannot access restricted directories on the system.
+For example the `NON_ROOT_USER` should only have read access to `<${branding}_HOME.
 ====
-If completing a non-standard installation, be aware that some applications depend on other ${branding} applications being installed.
 
-This hierarchy can be shown using the `app:tree` command
+==== Initial Startup
+
+Run the ${branding} using the appropriate script.
 
-[source]
+.*NIX
 ----
-${ddf-branding-lowercase}${at-symbol}local>app:tree
-+- opendj-embedded
-+- platform-app
-|   +- catalog-app
-|   |   +- search-ui-app
-|   |   |   +- spatial-app
-|   |   +- solr-app
-|   +- security-services-app
-|   |   +- admin-app
+<${branding}_HOME/bin/${branding-lowercase}
```

Comment:
Wouldn't one want to start ddf in the background? https://karaf.apache.org/manual/latest/